UI/src/app/home/home.component.html

```diff
@@ -1,88 +1,116 @@
-                [disabled]="shouldDisablePlotButton()">
-          <i class="fa fa-chart-line"></i> Plot
-        </button>
-      </div>
-    </div>
-    <div class="col-4 textinput">
-      <div class="row">
-        <app-text-input #textInputUpdate
-                        id="text-input-update"
-                        (plot)="updateGraph()"
-                        [isPlotTextInput]="false"
-                        [disabled]="disableUpdateTextField()" >
-        </app-text-input>
-      </div>
-      <div class="row button">
-        <button id="updateButton"
-                class="btn btn-md btn-warning"
-                type="button"
-                (click)="updateGraph()"
-                [disabled]="shouldDisableUpdateButton()">
-          <i class="fa fa-chart-line"></i> Update
-        </button>
-      </div>
+    <div class="row">
+        <div class="col-1">
+        </div>
+        <div class="col-6 textinput">
+            <app-text-input #textInput
+                            id="text-input"
+                            (plot)="plotGraph()"
+                            [isPlotTextInput]="true"
+                            [disabled]="false">
+            </app-text-input>
+        </div>
+        <div class="col-1" style="margin-left: 33px">
+            <div class="row button">
+                <button id="plotButton"
+                        class="btn btn-md btn-primary"
+                        type="button"
+                        (click)="plotGraph()"
+                        [disabled]="shouldDisablePlotButton()">
+                    <i class="fa fa-chart-line"></i> Plot
+                </button>
+            </div>
+        </div>
+        <div class="col-1" style="margin-left: 25px">
+            <div class="row button">
+                <button id="resetButton"
+                        class="btn btn-md btn-danger"
+                        type="button"
+                        (click)="clearAll()"
+                        [disabled]="shouldDisableClearButton()">
+                    <i class="fa fa-redo"></i> Clear
+                </button>
+            </div>
+        </div>
     </div>
-    <div class="col-2">
-
-    </div>
-
-  </div>
-  <hr>
-  <div class="row button">
-    <button id="resetButton"
-            class="btn btn-md btn-danger"
-            type="button"
-            (click)="clearAll()"
-            [disabled]="shouldDisableClearButton()">
-      <i class="fa fa-redo"></i> Clear
-    </button>
-  </div>
 </div>
-
-<div id="error" [hidden]="!err_">
-    <img src="assets/error.png" alt="Smiley face" >
+<hr>
+<div>
+    <div class="display-container">
+        <div id="error" [hidden]="!err_" class="msg-div">
+            <i class="fa fa-exclamation-triangle" style="font-size: 56px"></i>
+            <br>
+            <p>OOPS! Something went wrong.</p>
+        </div>
+        <div class="plot-div" [hidden]="err_">
+            <div [hidden]="!disableUpdateTextField()" class="msg-div">
```

Comment:
you are abusing those methodes: 
I'd like to have the two text fields to be shown next to each other the whole time, like it was before

other than that the design looks really nice

Reply:
I fixed it in #81 
maybe you take a look at it and merge it in here